packages/ghc/build.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+# Extract source tarball manually (avoids staging issues with hardlink entries in tarball)
+tar -xof ghc-$MINIMAL_ARG_VERSION-src.tar.xz
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)